### DISCUSSION TITLE
typos

### DISCUSSION THREAD

**Discussion by Akrepkapo** [IBAX-io/go-ibax#2085] [Typos （English）]
![IMG_20221222_014449](https://user-images.githubusercontent.com/96292241/209023655-3577aa83-a5bb-48a7-bdc5-fd2c28a35951.jpg)
![IMG_20221222_014617](https://user-images.githubusercontent.com/96292241/209023657-b1e7ca71-c13e-45c3-9640-04c12581eabc.jpg)
![IMG_20221222_014721](https://user-images.githubusercontent.com/96292241/209023659-b6389abf-a1e4-43a6-b095-0f6069fa9343.jpg)
![IMG_20221222_015035](https://user-images.githubusercontent.com/96292241/209023662-3b761862-7b19-48f7-84da-e2a986ab9a42.jpg)
![IMG_20221222_015518](https://user-images.githubusercontent.com/96292241/209023665-dd0f6b8f-bda2-4f39-b46e-9bf5612fd35d.jpg)

**Comment by zeland2323** [comment]
Thanks
